atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -361,7 +371,9 @@ private void performInternalCompactionIfNecessary(TableReference tableRef, Sweep
             Stopwatch watch = Stopwatch.createStarted();
             kvs.compactInternally(tableRef);
             long elapsedMillis = watch.elapsed(TimeUnit.MILLISECONDS);
-            log.debug("Finished performing compactInternally on {} in {} ms.", tableRef, elapsedMillis);
+            log.info("Finished performing compactInternally on {} in {} ms.",
```

Comment:
Leave at debug, compaction is know to not work, so no useful info here.. @ashrayjain for SA.